### ISSUE TITLE
A question about Setting 1

### ISSUE THREAD

**Issue by Junyu-Z** [bradyz/cross_view_transformers#34]
Thanks for your kindness in sharing your code with us!

I have a question about the Setting1.  According to the Table 1 in your paper, PON is under the Setting 1 that refers to the 100m x 50m at 25 cm resolution. The resolution of 25cm per pixel is mentioned in the 4.2 section of PON's paper, but I think the wording of 100m x 50m is not correct. According to PON's [code,](https://github.com/tom-roddick/mono-semantic-maps) pixel resolution of its BEV image is 196x200, so I think Setting 1 should refer to the 49m x 50m at 25cm resolution. Did I get something wrong?

**Comment by emilyemliyM** [comment]
> 

Hi , do you find the  generate_label.py?
thanks

**Comment by Junyu-Z** [comment]
> > 
> 
> Hi , do you find the generate_label.py? thanks

No, I directly use the provided map-view labels to train the model.
